Cargo.lock

```diff
@@ -1353,7 +1353,7 @@ dependencies = [
  "bitflags 2.6.0",
  "cexpr",
  "clang-sys",
- "itertools 0.11.0",
+ "itertools 0.10.5",
```

Comment:
not sure what renovate is cooking with these downgrades all the time. it probably doesn't actually hurt but it's just odd!

Reply:
i am pretty sure that once i merge this renovate will bring in a pr to update itertools.